packages/core/src/browser/preferences/preference-contribution.ts

```diff
@@ -115,6 +107,9 @@ export class PreferenceSchemaProvider extends PreferenceProvider {
     @inject(PreferenceConfigurations)
     protected readonly configurations: PreferenceConfigurations;
 
+    @inject(PreferenceLanguageOverrideService)
+    protected readonly preferenceOverrideService: PreferenceLanguageOverrideService;
```

Comment:
I think this is redundant since the PreferenceProvider already has access to this service